checks/shell_download_validate.go

```diff
@@ -575,6 +575,42 @@ func isPipUnpinnedDownload(cmd []string) bool {
 	return false
 }
 
+func isChocoUnpinnedDownload(cmd []string) bool {
```

Comment:
Can you please write tests for this?

Reply:
@Alan-Jowett can you:
1. Can you add one more unpinned command in this file? Just to check that the code properly finds multiple occurrences
2. Add some example for function `TestShellscriptInsecureDownloadsLineNumber`
3. Add some examples in ./testdata/Dockerfile-pkg-managers
4. Add some examples in ./testdata/script-pkg-managers